application/modules/apiV2/controllers/WorkflowController.php

```diff
@@ -0,0 +1,210 @@
+{
+
+    /**
+     * @OA\Post(
+     *     path="/workflow",
+     *     tags={"workflow"},
+     *     summary="Execute workflow",
+     *     description="Execute a workflow",
+     *     operationId="execute",
+     *     @OA\Parameter(
+     *         name="execute",
+     *         in="query",
+     *         description="name of workflow",
+     *         required=true,
+     *         @OA\Schema(
+     *             type="string",
+     *         )
+     *     ),
+     *     @OA\RequestBody(
+     *          required=false,
+     *          @OA\MediaType(
+     *              mediaType="application/json",
+     *              @OA\Schema(
+     *                  type="object",
+     *                  required={},
+     *                  @OA\Property(
+     *                      type="object",
+     *                      property="workflow",
+     *                      description="workflow name",
+     *                      @OA\Property(
+     *                          type="object",
+     *                          property="params",
+     *                          description="workflow params",
+     *                              example={ "ci_id": 12 }
+     *                      ),
+     *                      @OA\Property(
+     *                          type="string",
+     *                          property="startAt",
+     *                          description="start at ISO8601",
+ *                              example="2004-02-12T15:19:21+00:00"
+     *                      ),
+     *                      @OA\Property(
+     *                          type="bool",
+     *                          property="forceAsync",
+     *                          description="start the workflow always async. _false_ can't override async workflows to be run sync",
+ *                              example=false
+     *                      ),
+     *                  ),
+     *              ),
+     *          ),
+     *     ),
+     *     @OA\Response(
+     *         response=200,
+     *          description="CI saved successfully",
+     *          @OA\MediaType(
+     *              mediaType="application/json",
+     *              @OA\Schema(
+     *                  allOf={
+     *                      @OA\Schema(ref="#/components/schemas/Util_Response"),
+     *                  },
+     *                  @OA\Property(
+     *                      property="message",
+     *                      type="string",
+     *                      example="Workflow executed successfully",
+     *                  ),
+     *                  @OA\Property(
+     *                      property="data",
+     *                      type="object",
+     *                      example="[ { ""ci_id"": 1123 }, { ""ci_id"": 1124 } ]",
+     *                  ),
+     *              ),
+     *          ),
+     *     ),
+     *     @OA\Response(
+     *          response=400,
+     *          description="Workflow failed",
```

Comment:
400 Bad Request shouldn't be used for failed workflows.
`The client SHOULD NOT repeat the request without modifications.`
This is not the case here.

The client did nothing wrong. This is an error that should be resolved on the server side, so 500 Internal Server Error is more appropriate here.

The status code should still be used and documented for request validations like incorrect `startAt` format.